File: 03.business-os/NameCardManager/android/app/src/main/java/com/lsam/NameCardManager/runtime/commonos/bootstrap/CommonOSBootstrapRegistry.java
package com.lsam.NameCardManager.runtime.commonos.bootstrap;

public final class CommonOSBootstrapRegistry {

    public CommonOSBootstrapDescriptor createDefault() {
        return new CommonOSBootstrapDescriptor(
                "NameCardManager",
                "commonos_bootstrap",
                "CommonOS",
                "NameCardManager",
                "runtime/commonos",
                "runtime/commonos/entry",
                "runtime/commonos/contract"
        );
    }
}

File: 03.business-os/_archive/commonos_legacy_skeleton_20260425_184835/app_local/NameCardManager/iphone/NameCardManager/CommonOSBootstrap/CommonOSBootstrapRegistry.swift
import Foundation

enum CommonOSBootstrapRegistry {
    static func createDefault() -> CommonOSBootstrapDescriptor {
        CommonOSBootstrapDescriptor(
            appName: "NameCardManager",
            launchMode: "commonos_bootstrap",
            uiOwner: "CommonOS",
            businessCanonOwner: "NameCardManager",
            runtimeRoute: "runtime/commonos",
            entryRoute: "runtime/commonos/entry",
            contractRoute: "runtime/commonos/contract"
        )
    }
}
